crates/tabby-common/src/index/code/mod.rs

```diff
@@ -49,10 +50,18 @@ pub fn body_query(tokens: &[String]) -> Box<dyn Query> {
 }
 
 fn filepath_query(filepath: &str) -> Box<TermQuery> {
```

Comment:
if you invoke normalize_path in this function, you don't have to modify `api/code.rs` nor `code_search_query` function.